scripts/Makefile.depends

```diff
@@ -0,0 +1 @@
+# autogenerated
```

Comment:
Accidentally added?

Reply:
shoot, yes. Should I just delete the file?

Reply:
Yeah. If you're feeling picky you can rebase it out and force-push, but for this small a file it's not at all necessary